### ISSUE TITLE
Feature: Manual input

### ISSUE THREAD

**Issue by eltyBelgium** [mariomka/vue-datetime#171]
## Versions
- vue: 2.6.10
- vue-datetime: 1.0.0-beta.10
- luxon: 1.17.2

### Description: 
possible way to do a manual input in the box. That the parsing happens in the datepicker too

**Comment by Herz3h** [comment]
This would be really useful, I'll try to look into it

**Edit**: After attempting to implement this, there is some limitation by using luxon, input date cannot really be validated since luxon doesn't parse custom formats... Also the library internally use a datetime string to store datetime/date/time regardless of mode. This means it will be hard to parse, say a custom format like DD/MM/YYYY and merge it back into the datetime string dynamically.
